<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointView</class>
 <widget class="QWidget" name="PointView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>75</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="xLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="yLabel">
     <property name="text">
      <string>y</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="zLabel">
     <property name="text">
      <string>z</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="zLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
